lib/internal/streams/legacy.js

```diff
@@ -111,4 +112,6 @@ function prependListener(emitter, event, fn) {
     emitter._events[event] = [fn, emitter._events[event]];
 }
 
+Stream.defaultHighwaterMark = Buffer.poolSize;
```

Comment:
We need a few tests

Reply:
If the value of `Buffer.poolSize` is modified, this will not be modified, is this voluntary?